lib/logger/utils.ts

```diff
@@ -113,9 +114,7 @@ export function sanitizeValue(_value: unknown, seen = new WeakMap()): any {
     value = prepareError(value);
   }
 
-  const valueType = typeof value;
-
-  if (value && valueType !== 'function' && valueType === 'object') {
+  if (value && typeof value !== 'function' && typeof value === 'object') {
```

Comment:
Shouldn't typescript already be able to track that?

@JamieMagee ideas?

Reply:
Fixed here: https://github.com/renovatebot/renovate/pull/12859